tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -35,7 +35,6 @@ Here is a complete class example based on these guidelines:
     ## Initializes states and delegates engine callbacks ([method Node._physics_process],
     ## [method Node._unhandled_input]) to the state.
 
-
```

Comment:
This seems unrelated and breaks the general style which has two lines between entries

Reply:
For this particular case, there isn't a class or function just below (but a variable with a setter), so I don't think two blank lines are needed here.